### DISCUSSION TITLE
Error caused by vim.treesitter.start when opening files from NetRW

### DISCUSSION THREAD

**Discussion by ghost** [nvim-treesitter/nvim-treesitter#4062] [Q&A] [answered]
Hello :wave: 

I'm seeing the following error message when opening files from NetRW after updating treesitter.

```
Error detected while processing function <SNR>42_NetrwBrowseChgDir[197]..BufReadPost Autocommands for "*.lua"..FileType Autoc
ommands for "*":
Error executing lua callback: .../start/nvim-treesitter/lua/nvim-treesitter/highlight.lua:26: attempt to call field 'start' (
a nil value)
stack traceback:
        .../start/nvim-treesitter/lua/nvim-treesitter/highlight.lua:26: in function 'attach'
        ...er/start/nvim-treesitter/lua/nvim-treesitter/configs.lua:505: in function 'attach_module'
        ...er/start/nvim-treesitter/lua/nvim-treesitter/configs.lua:528: in function 'reattach_module'
        ...er/start/nvim-treesitter/lua/nvim-treesitter/configs.lua:131: in function <...er/start/nvim-treesitter/lua/nvim-tr
eesitter/configs.lua:130>
```

I browsed through the recent commits and found [this](https://github.com/nvim-treesitter/nvim-treesitter/commit/cf6056489135f22dfe00b50b34e41439e2ea72dd) one which seems to have introduced the change that my environment is failing on by changing `M.start` to `vim.tresitter.start`.

I'm asking this in Q&A before creating an bug issue because I'm quite new to Treesitter, and assuming the problem is on my end.

NVIM Version
```
NVIM v0.8.2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by runner@fv-az190-151
```

Any help would be greatly appreciated.
Thanks.

**Comment by clason** [comment, marked as answer]
What does `:lua =vim.treesitter.start` say?

**Reply by ghost** [reply]
It returns `nil`

**Reply by clason** [reply]
Then your Neovim installation is broken (it can't find its runtime files).

If there's _any_ caching in your config (impatient, hotpot, lazy, whatever), make sure to wipe all caches after uprading to a new version.

**Reply by ghost** [reply]
Thank you for the help! That fixed it.

**Reply by szy-kenn** [reply]
I'm having the same problem, how do I wipe all caches in my config?
